installer/charts/rhtap-dh/templates/extra-env.yaml

```diff
@@ -28,6 +28,16 @@ data:
     ARTIFACTORY__API_TOKEN: {{ $artifactorySecretData.token }}
     ARTIFACTORY__URL: {{ $artifactorySecretData.url }}
 {{- end }}
+{{- $azureSecretObj := (lookup "v1" "Secret" $integrationNamespace "rhtap-azure-integration") }}
+{{- $azureSecretData := ($azureSecretObj.data | default dict) }}
+{{- if $azureSecretData }}
+    AZURE__HOST : {{ $azureSecretData.host }}
+    AZURE__ORG : {{ $azureSecretData.organization }}
+    AZURE__TOKEN: {{ $azureSecretData.token }}
+    AZURE__CLIENT__ID: {{ $azureSecretData.clientId }}
```

Comment:
Since auth is optional, those vars might not be set, maybe put some guard around them?